### ISSUE TITLE
Cronjob model click doesn't working when method is in extends class

### ISSUE THREAD

**Issue by ihor-sviziev** [enriquepiatti/Magicento#104]
When I trying to click on my 'my_module/my_model_custom::run' line - nothing happens, but if i change it to "my_module/my_model_custom::run2" - phpstorm open file with this method.

In my config.xml i have: 

``` xml
    <crontab>
        <jobs>
            <my_module>
                <schedule>
                    <config_path>my_module/.../cron_expr</config_path>
                </schedule>
                <run>
                    <model>my_module/my_model_custom::run</model>
                </run>
            </my_module>
        </jobs>
    </crontab>
```

Class My_Module_Model_My_Model_Abstract:

``` php
class My_Module_Model_My_Model_Abstract
{
    public function run()
    {
        ...
    }
}
```

Class My_Module_Model_My_Model_Custom:

``` php
class My_Module_Model_My_Model_Custom extends My_Module_Model_My_Model_Abstract
{
    public function run2() {
        ...
    }
}
```

Magicento version: 2.6.0

**Comment by enriquepiatti** [comment]
This is because the method is defined inside a parent class. I've fixed it thanks for reporting it!
I will release a new version with this fix and some others during this week.

**Comment by ihor-sviziev** [comment]
Received new update today, but this issue isn't fixed there

**Comment by enriquepiatti** [comment]
Igor, the new update was not released yet, maybe you have received v2.6.0 (the fix will be in 2.6.2+)

**Comment by ihor-sviziev** [comment]
@enriquepiatti thanks.
